Revit_Engine/Query/GetRevitParameterValue.cs

```diff
@@ -59,6 +60,27 @@ public static object GetRevitParameterValue(this IBHoMObject bHoMObject, string
                     return param.Value;
             }
 
+            RevitIdentifiers identifierFragment = bHoMObject.Fragments?.FirstOrDefault(x => x is RevitIdentifiers) as RevitIdentifiers;
+            if (identifierFragment != null)
+            {
+                string paramName = string.Concat(parameterName.Where(c => !char.IsWhiteSpace(c)));
+                if (Reflection.Query.PropertyNames(identifierFragment).Contains(paramName))
+                    return Reflection.Query.PropertyValue(identifierFragment, paramName);
+            }
+
+            List<object> bHoMPropList = Reflection.Query.PropertyObjects(bHoMObject);
+            List<IBHoMObject> bHoMProps = bHoMPropList.OfType<IBHoMObject>().ToList();
+            foreach (IBHoMObject bHoMProp in bHoMProps)
+            {
+                RevitPulledParameters typePullFragment = bHoMProp.Fragments?.FirstOrDefault(x => x is RevitPulledParameters) as RevitPulledParameters;
+                if (typePullFragment?.Parameters != null)
+                {
+                    RevitParameter param = typePullFragment.Parameters.FirstOrDefault(x => x.Name == parameterName);
+                    if (param != null)
+                        return param.Value;
```

Comment:
Could we please record a warning here saying that the value for an object under BHoM_Guid X has been retrieved from parameters belonging to its property Y?